Wikipedia/Code/WebViewController.m

```diff
@@ -967,11 +1006,11 @@ - (void)scrollTappedReferenceUpIfNecessaryWithReferencePageViewController:(WMFRe
             }
             CGFloat delta = self.webView.scrollView.contentOffset.y - newOffsetY;
             CGPoint centeredOffset = CGPointMake(self.webView.scrollView.contentOffset.x, newOffsetY);
-            [self.webView.scrollView wmf_safeSetContentOffset:centeredOffset
-                                                     animated:YES
-                                                   completion:^(BOOL finished) {
-                                                       controller.backgroundView.clearRect = CGRectOffset(windowCoordsRefGroupRect, 0, delta);
-                                                   }];
+            [self scrollToOffset:centeredOffset
+                        animated:YES
+                      completion:^{
+                          controller.backgroundView.clearRect = CGRectOffset(windowCoordsRefGroupRect, 0, delta);
```

Comment:
I can repro this in develop as well - the clearRect is off on iPhone (tested in iOS13). I can file a separate bug if necessary.

![IMG_0012](https://user-images.githubusercontent.com/3620196/64873402-950c2c80-d60e-11e9-9f1b-32aeb242f77f.PNG)

Reply:
If that repros on develop it'd be better to file separately. Thanks!